src/components/pages/Story1/index.tsx

```diff
@@ -1,33 +1,52 @@
 import React from 'react';
 import { useHistory } from 'react-router';
+import Wireframe2 from '../../../assets/neutral_wireframe.png';
 import Wireframe from '../../../assets/wireframe.png';
 import Layout, { SceneProps } from '../../shared/Layout';
+import s1s1_neutral from '../../sketches/s1s1_neutral';
```

Comment:
Let's just call this Scene 1 and put the file in a Story1 folder?

Reply:
Ok sounds good